gatsby-node.js

```diff
@@ -11,21 +11,22 @@ const { defaultLocale, supportedLocales } = config.siteMetadata
 
 exports.onPreBootstrap = () => {
   const base = parseBaseConfig()
-  const pages =
-    base.collections.find(c => c.name === "pages").files.map(f => f.file)
+  const fileCollections =
+    base.collections.filter(c => c.name === "pages" || c.name === "header")
+  const files = fileCollections
+    .reduce((acc, curr) => acc.concat(curr.files.map(f => f.file )), [])
```

Comment:
This can just be `fileCollections.flatMap(_ => _.files.map(f => f.file))`, I think.

Reply:
Oo. If it works I'll make the change in an upcoming PR!

Reply:
In the dept of code golf: `fileCollections.flatMap({ files } => files.map({ file } => file))`. Clarity win is... Very arguable.